stdlib/public/runtime/ImageInspectionCOFF.h

```diff
@@ -32,34 +32,11 @@ struct SectionInfo {
 };
 
 static constexpr const uintptr_t CurrentSectionMetadataVersion = 1;
-
-struct MetadataSections {
-  uintptr_t version;
-  uintptr_t reserved;
-
-  mutable const MetadataSections *next;
-  mutable const MetadataSections *prev;
-
-  struct Range {
-    uintptr_t start;
-    size_t length;
-  };
-
-  Range swift5_protocols;
-  Range swift5_protocol_conformances;
-  Range swift5_type_metadata;
-  Range swift5_typeref;
-  Range swift5_reflstr;
-  Range swift5_fieldmd;
-  Range swift5_assocty;
-  Range swift5_repl;
-  Range swift5_reps;
-};
 } // namespace swift
 
 // Called by injected constructors when a dynamic library is loaded.
 SWIFT_RUNTIME_EXPORT
-void swift_addNewDSOImage(const void *addr);
+void swift_addNewDSOImage(void *addr);
```

Comment:
I don't think it's possible to drop 'const' here, as users might get 'dropping const qualifier' compile errors.